src/Data/List/Relation/Unary/Any/Properties.agda

```diff
@@ -293,6 +306,11 @@ module _ {f : A → B} where
          Any (P ∘ f) xs ↔ Any P (List.map f xs)
   map↔ = inverse map⁺ map⁻ (map⁻∘map⁺ _) map⁺∘map⁻
 
+module _ {a b p q} {A : Set a} {B : Set b} {f : A → B}
+         {P : A → Set p} {Q : B → Set q} where
+  gmap : P ⋐ Q ∘ f → Any P ⋐ Any Q ∘ map f
```

Comment:
Here you can drop `{a b p q} {A : Set a} {B : Set b}` thanks to the [`variables`](https://github.com/agda/agda-stdlib/blob/40bd22418fcfb47d0d620f41dd5495c76fb38c04/src/Data/List/Relation/Unary/Any/Properties.agda#L52)
introduced at the top of the file.

Reply:
Done